package.json

```diff
@@ -28,12 +28,11 @@
     "docs:lint": "prettier '**/*.md' --check",
     "docs:lint:fix": "prettier '**/*.md' --write",
     "test": "lerna run test --concurrency 1",
-    "test:unit": "lerna run test:unit --concurrency 1",
-    "test:browsers": "lerna run test:browsers --concurrency 1",
-    "test:e2e": "lerna run test:e2e --concurrency 1",
-    "test:e2e:sim": "lerna run test:e2e:sim",
```

Comment:
Do we not have this in any packages anymore?

Reply:
No this one never existed since I started looking into tests.